Adds the following items:

Bunnky opened the pull request:
Ramen Beef
Ramen Chicken
Ramen Seafood
Ramen Beef (Hearty)
Ramen Chicken (Hearty)
Ramen Seafood (Hearty)
Bagel
Bagel Bite (Special) (Needs More)
Bagel with Cream Cheese
Meatloaf
Sausage
Soylent
Tater Tots
Green Onion
Ketchup
Mustard
Cream Cheese
Pepper + Bush
Green Onion + Bush

Sefiraat commented:
Yeah as you mentioned in Discord, your formatting's a bit whack - the content + the build looks valid at a glance. We will work through formatting on Discord until its resolved before I take a fuller look!

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            simplePlayerEffect(
+                player,
+                999,
+                new Pair<>(PotionEffectType.HEALTH_BOOST, 4),
+                new Pair<>(PotionEffectType.SPEED, 3)
```
```suggestion
                new Pair<>(PotionEffectType.HEALTH_BOOST, 1),
                new Pair<>(PotionEffectType.SPEED, 0)
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            simplePlayerEffect(
+                player,
+                999,
+                new Pair<>(PotionEffectType.HEALTH_BOOST, 4),
+                new Pair<>(PotionEffectType.SPEED, 3)
```
```suggestion
                new Pair<>(PotionEffectType.HEALTH_BOOST, 1),
                new Pair<>(PotionEffectType.SPEED, 0)
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java:
```diff
@@ -3401,5 +3465,149 @@ private CultivationStacks() {
+        "A filling choice.",
+        "",
+        Theme.CLICK_INFO.asTitle("Healing", "Full"),
+        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 5"),
+        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 4")
```
```suggestion
        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 2"),
        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 1")
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java:
```diff
@@ -3401,5 +3465,149 @@ private CultivationStacks() {
+        "A satisfying choice.",
+        "",
+        Theme.CLICK_INFO.asTitle("Healing", "Full"),
+        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 5"),
+        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 4")
```
```suggestion
        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 2"),
        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 1")
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            simplePlayerEffect(
+                player,
+                999,
+                new Pair<>(PotionEffectType.HEALTH_BOOST, 4),
+                new Pair<>(PotionEffectType.SPEED, 3)
```
```suggestion
                new Pair<>(PotionEffectType.HEALTH_BOOST, 1),
                new Pair<>(PotionEffectType.SPEED, 0)
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            null, Ingredients.CREAM_CHEESE.getItem(), null,
+            null, null, null
+        },
+        player -> simplePlayerEffect(player, 10,
+                                     new Pair<>(PotionEffectType.HEALTH_BOOST, 0)
+        )
```
```suggestion
        player -> simplePlayerEffect(player, 10, new Pair<>(PotionEffectType.HEALTH_BOOST, 0))
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            SlimefunItems.SALT, Products.ONION.getChoppedItem(), Ingredients.MUSTARD.getItem()
+        },
+        player -> simplePlayerEffect(player, 999,
+                                     new Pair<>(PotionEffectType.HEALTH_BOOST, 5),
+                                     new Pair<>(PotionEffectType.DAMAGE_RESISTANCE, 4),
+                                     new Pair<>(PotionEffectType.FAST_DIGGING, 3),
+                                     new Pair<>(PotionEffectType.SLOW, 2)
```
```suggestion
                                     new Pair<>(PotionEffectType.HEALTH_BOOST, 3),
                                     new Pair<>(PotionEffectType.DAMAGE_RESISTANCE, 2),
                                     new Pair<>(PotionEffectType.FAST_DIGGING, 1),
                                     new Pair<>(PotionEffectType.SLOW, 1)
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java:
```diff
@@ -3401,5 +3465,149 @@ private CultivationStacks() {
+        "Meatloaf",
+        "Nothing like it! Let's go.",
+        "",
+        Theme.CLICK_INFO.asTitle("Healing", "Full"),
+        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 6"),
+        Theme.CLICK_INFO.asTitle("Effect 2", "Damage Reduction + 5"),
+        Theme.CLICK_INFO.asTitle("Effect 3", "Haste + 4"),
+        Theme.CLICK_INFO.asTitle("Effect 4", "Slowness + 3")
```
```suggestion
        Theme.CLICK_INFO.asTitle("Healing", "Full"),
        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 4"),
        Theme.CLICK_INFO.asTitle("Effect 2", "Damage Reduction + 3"),
        Theme.CLICK_INFO.asTitle("Effect 3", "Haste + 2"),
        Theme.CLICK_INFO.asTitle("Effect 4", "Slowness + 2")
```

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            Products.AVOCADO.getBlendedItem(), new ItemStack(Material.WATER_BUCKET), Products.VANILLA.getBlendedItem(),
+            Products.SPINACH.getBlendedItem(), Products.SOY_BEANS.getBlendedItem(), new ItemStack(Material.ICE)
+        },
+        player -> simplePlayerEffect(player, 0,
```
Bring player, 0 and each pair onto their own lines

Sefiraat commented on src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
+            null, Products.POTATO.getFriedItem(), null,
+            null, null, null
+        },
+        player -> simplePlayerEffect(player, 1,
+                                     new Pair<>(PotionEffectType.SPEED, 1)
+        )
```
```suggestion
        player -> simplePlayerEffect(player, 1, new Pair<>(PotionEffectType.SPEED, 1))
```

Bunnky commented:
Not sure what I did up there xD however I have included the suggestions, removed an unused Skin, and added an Import for ParticleUtils. I tried to do everything through the IDE because I thought I could, but I couldn't figure it out so ignore the goofs lol :)

Bunnky commented:
@J3fftw1

Sefiraat reviewed (approved):
Looking good to me!

Sefiraat commented:
> @J3fftw1 

That's not how you request a review, I'll send you some screenshots when I'm at my PC, I've added Jeff for ya :)

J3fftw1 commented on src/main/java/dev/sefiraat/cultivation/api/utils/Skins.java:
```diff
@@ -507,7 +507,31 @@ public enum Skins {
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/8748-bagel
+    BAGEL("502e92f13de3bee69228c384478e761230681e5fce9bda195daeaf8484139331"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/8750-poppyseed-bagel
+    BAGEL_EVERYTHING("653aaae67dda225b78e82886d046c67fdf37f1bbe2dfc55f1324781f8159f5"),
```
Bagel everything isn’t that descriptive

Sefiraat replied:
Everything bagel is, annoyingly, the name of a type of bagel, and marking up with the archetype first followed by the descriptor is in line with my code style, so while it seems to not make sense it annoyingly does. everything Bagel, 'murica
